Assignment/Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
-   }
-}
\ No newline at end of file
+﻿//Swap two Number from the given number list
+Random rnd = new Random ();
+int input = rnd.Next (5, 10);
+Console.Write ("Input numbers : ");
+List<int> numbers = new List<int> ();
+for (int i = 0; i < input; i++) {
+   int n = rnd.Next (1, 10);
+   numbers.Add (n);
+   Console.Write (n + " ");
+}
+Console.WriteLine ($"\nSwap two index values between 1 to {input}");

```

Comment:
You are prompting for index but expecting natural value. Can you change appropriately?